data/product.d/fedora-server.conf

```diff
@@ -15,6 +15,7 @@ file_system_type = xfs
 default_partitioning =
     /    (min 2 GiB, max 15 GiB, encrypted)
     swap (encrypted)
+default_scheme = LVM
```

Comment:
I would move this line above `default_partitioning` to improve the readability. Same for Fedora Atomic Host.

Reply:
Done, thanks for the suggestion. 👍